src/js/media-chrome-range.ts

```diff
@@ -240,13 +240,13 @@ template.innerHTML = /*html*/ `
     <div id="startpoint"></div>
     <div id="endpoint"></div>
     <div id="appearance" part="appearance">
```

Comment:
Is `appearance` an undocumented part? Or should it not be exposed? It's kind of an odd concept.

Reply:
I tend to agree now but it was documented unfortunately so I don't think we can remove it :(

Reply:
I know we don't frequently do this, but I think this is a good case to deprecate (and document the deprecation). I think several of these names may be confusing or at least not intuitive. The track-specific parts are not obviously track specific. I'm not sure why the div indicating what's been buffered is called `"highlight"`? I don't have a clear sense of what `"pointer"` actually is from the name or the description? Might be worth some time/thought/discussion getting these names right, since, per @luwes's last comment, once we add these parts + names it's a bigger deal to change or remove them.

If we _**do**_ deprecate `"appearance"` and decide to rename it in the deprecation process, I think we can just:
1. add the new part name to the attribute
2. update the docs indicating that appearance is deprecated and will be removed in a future release and they should use whatever the new part name is (which should also be documented).

Reply:
Sounds good yes, I don't want to rush this. 

There's much to think about also heading in the future CSS vars vs CSS parts. And if we ever want to deprecate CSS vars in favor of a CSS part.

`highlight` is called like that because it's in the base class MediaChromeRange. 
It doesn't really know about buffering or even if it's a time range.

the appearance naming is from CSS and webkit uses this as well. it's the visual UI, the appearance :)